code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -63,7 +63,7 @@
 		to_chat(user, "<span class='warning'>This syringe is broken!</span>")
 		return
 
-	if(user.a_intent == I_HURT && ismob(target))
+	if(user.a_intent == I_HURT && ismob(target) && !isrobot(user))

```

Comment:
this literally makes robots unable to do syringe stabs lmao what is this entire pr

Reply:
Right?

Kinda frustrated with this. Just make robot syringes unbreakable, don't fucking make them **unable to stab.**

Reply:
I'd say make them breakable but fix them automatically when entering a charger or something like that.
Since stabbing can be kinda strong depending on how its used.